github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-postsubmits.yaml

```diff
@@ -0,0 +1,73 @@
+postsubmits:
+  kubevirt/kubevirt:
+  - name: push-latest-kubevirt-images
```

Comment:
I think we can drop this job too. These images were pushed on travis but actually never used. Instead we have a periodic which pushes daily developer builds and does manifest releases on gcs: https://github.com/kubevirt/project-infra/blob/58a2a1a8735b56efd5983535bd9134dc39f57037/github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml#L135

Reply:
Let's instead reconfigure the daily periodic to push to quay.

Reply:
Done, successful test build here https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/logs/periodic-kubevirt-push-nightly-build-master/1334449652883263488 this is how one of the images uploaded by the nightly periodic looks like https://quay.io/repository/kubevirt/alpine-container-disk-demo?tab=tags

Reply:
Deployed the developer build from quay, started a test VM and verified that all images were coming from quay. No issues detecte.